design/local-node-agent.md

```diff
@@ -91,17 +107,72 @@ The experience is also normalized across other distributed storage platforms tha
 Instead of the user having to create a storageclass specific for the underlying distributed storage (e.g., RBD if Rook is using Ceph), this will abstract that decision away, allowing the user to simply request storage.
 This was already true of the hybrid approach in Rook v0.5, but it's worth noting here still.
 
+#### Volume Attach CRD Details
+This section will discuss the volume attach CRD in more detail.
+The name (primary identity) of each CRD instance will be the PVC name that it is for.
```

Comment:
nit: To be correct, the name of the CRD is the PV name. Even though, it is the same as PVC., this brings a settle distinction between PVs and PVCs.

Reply:
that actually makes me feel better that it is the name of the PV, that feels more in line with what object in the system we are actually trying to track/capture. I will update this in the doc.